.travis.yml

```diff
@@ -163,7 +163,7 @@ script:
   # Make sure that relaxed strides checking is actually in effect; otherwise fail loudly
   - if [ "$NPY_RELAXED_STRIDES_CHECKING" == "1" ]; then python -c'import numpy as np; assert np.ones((10,1),order="C").flags.f_contiguous'; fi
   # Test that mpmath actually uses gmpy2
-  - python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
+  #- python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
```

Comment:
Could we just delete this and the above line then or is the anecdotal information sufficiently important to leave in source?

Reply:
May also be worth checking for i.e., `gympy2-2.0.8` before deleting as I note above, if that's not a crazy idea?

Reply:
Apparently the reason for having this `mpmath` backend is solely for speed in testing. With that in mind, this would indicate that the assertion is not strictly needed. Rather than remove it entirely I thought it would be wiser to leave it in as a comment, in case it needed to be resurrected.
However, I'm not thinking it would be better to leave it in for the linux build, but not for the macOS build.